extra_extra.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
   s.test_files = Dir["spec/**/*"]
 
-  s.add_runtime_dependency "rails", "~> 5"
+  s.add_runtime_dependency "rails", ">= 4", "~> 5"
```

Comment:
@davetron5000 A little different than initially described, I changed the `<= 5` to `~> 5` so updates to Rails 5 will get installed.